spec/spec_helper.rb

```diff
@@ -1,46 +1,64 @@
-require 'rubygems'
-require 'bundler/setup'
 require 'rspec'
-require 'fake_dynamo'
-require 'aws-sdk'
 
-Aws.config.update({
+require 'aws-sdk-dynamodb'
+require 'dynamodb-mutex'
+
+Aws.config.update(
   credentials: Aws::Credentials.new(
     'your_access_key_id',
     'your_secret_access_key'
   ),
   endpoint: 'http://localhost:4567'
-})
+)
 
-require 'dynamodb-mutex'
+AMAZON_LOCAL_DYNAMODB_URL =
+  'https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/dynamodb_local_latest.tar.gz'.freeze
+DYNAMODB_JAR_DIR = 'resources'.freeze
+DYNAMODB_TMP_PATH = '/tmp/dynamodb_local_latest.tar.gz'.freeze
+
+def fetch_amazon_dynamodb_local
+  Dir.mkdir(DYNAMODB_JAR_DIR) unless Dir.exist?(DYNAMODB_JAR_DIR)
+
+  return if File.exist?("./#{DYNAMODB_JAR_DIR}/DynamoDBLocal.jar")
+
+  raise "Error downloading #{AMAZON_LOCAL_DYNAMODB_URL}" unless \
+    system("wget #{AMAZON_LOCAL_DYNAMODB_URL} -O #{DYNAMODB_TMP_PATH} -q")
+
+  raise "Error unpacking #{DYNAMODB_TMP_PATH}" unless \
+    system("tar -xf #{DYNAMODB_TMP_PATH} -C #{DYNAMODB_JAR_DIR}")
```

Comment:
`tar -xf` vs. `tar xf`?

Reply:
I don't have a (really) strong opinion on this, but for me it's a question of consistency.
I like my CLI arguments to be GNU style and for that reason add the dash by default.